### PR TITLE
[3.5] supervision maintenance unttled

### PR DESCRIPTION
**Pull request by kvahed**
https://jenkins01.arangodb.biz/view/PR/job/arangodb-matrix-pr/7875/

### CODE REVIEW

**Comment by KVS85** [comment]
Tests blue except known failure on Windows (https://jenkins01.arangodb.biz/job/arangodb-matrix-pr-windows/1866/EDITION=enterprise,STORAGE_ENGINE=rocksdb,TEST_SUITE=single,limit=windows&&test/artifact/testfailures.log):
```
* Test "active_failover"
    [FAILED]  tests\js\client\active-failover\basic.js

      "testFollowerSelection" failed: at assertion #193: Leader has too little or too many documents: (false) does not evaluate to true
(Error
    at assertTrue (d:\hw15-windows\oskar\work\ArangoDB\js\common\modules\jsunity\jsunity.js:72:19)
    at Object.testFollowerSelection (d:\hw15-windows\oskar\work\ArangoDB\tests\js\client\active-failover\basic.js:431:7)
    at Object.run (d:\hw15-windows\oskar\work\ArangoDB\js\common\modules\jsunity\jsunity.js:532:23)
    at Object.Run [as run] (d:\hw15-windows\oskar\work\ArangoDB\js\common\modules\jsunity.js:195:24)
    at d:\hw15-windows\oskar\work\ArangoDB\tests\js\client\active-failover\basic.js:525:9
    at d:\hw15-windows\oskar\work\ArangoDB\tests\js\client\active-failover\basic.js:528:3
    at RunTest (d:\hw15-windows\oskar\work\ArangoDB\js\common\modules\jsunity.js:275:12)
    at d:\hw15-windows\oskar\work\ArangoDB\js\common\modules\@arangodb\testrunner.js:66:13
    at arrayEach (d:\hw15-windows\oskar\work\ArangoDB\js\node\node_modules\lodash\lodash.js:516:11)
    at Function.forEach (d:\hw15-windows\oskar\work\ArangoDB\js\node\node_modules\lodash\lodash.js:9344:14))


      "totalSetUp" failed: undefined


      "totalTearDown" failed: undefined



   Suites failed: 1 Tests Failed: 3
```

**Comment by kvahed** [comment]
https://jenkins01.arangodb.biz/view/PR/job/arangodb-matrix-pr/7893/

File: CHANGELOG [hunk 1]
```diff
@@ -1,11 +1,10 @@
 v3.5.5 (XXXX-XX-XX)
 -------------------
-
-* Make sure heartbeats are actually sent out and received every second.
-
+* Supervision maintenance mode from "on" and TTL to expiry string.
 
 v3.5.4 (2020-01-22)
 -------------------
+* Make sure heartbeats are actually sent out and received every second.
```
Comment (neunhoef) [inline CHANGELOG]: This is wrong, this will only appear in 3.5.5, please move up to 3.5.5

**Comment by kvahed** [comment]
http://jenkins.arangodb.biz:8080/job/arangodb-matrix-pr/8231/

**Comment by KVS85** [comment]
Tests blue!

**Pull request merged**